### DISCUSSION TITLE
Pretrained model

### DISCUSSION THREAD

**Discussion by marcomiglionico94** [Raschka-research-group/coral-cnn#41] [General]
Hi, do you have any pretrained model so that it is possible to do inference using the network, instead of training everything from scratch?
Thanks

**Comment by rasbt** [comment]
in the beginning, i saved all the networks but due to the large file size, I later just saved the predictions and probabilities on the test set. I think i should still have them on my external backup drive. which one are you interested in? Btw training takes "only" 1-4 hours depending on the dataset.

**Comment by rasbt** [comment]
Just found them. You can get the pt files from here: https://drive.google.com/drive/folders/199T9UZkMgaRSikTW40J0KH-0pJwzilpK?usp=sharing

(they may only work for PyTorch 1.0 though).

**Comment by marcomiglionico94** [comment]
Thanks!! @rasbt
